dowhy/utils/plotting.py

```diff
@@ -199,3 +199,9 @@ def _plot_as_pyplot_figure(pygraphviz_graph: Any, figure_size: Optional[Tuple[in
 
         if figure_size is not None:
             plt.rcParams["figure.figsize"] = org_fig_size
+
+def pretty_print_graph(graph):
```

Comment:
rather than # comment, add a docstring. That is automatically picked up by sphinx docs generator.